lib/core/include/irods/rodsError.h

```diff
@@ -33,7 +33,10 @@ typedef struct ErrorStack {
     struct ErrorMessage **errMsg;
 } rError_t;
 
-void allocateRErrorMsgIfNecessary(struct ErrorStack** myError);
+/// \brief Allocate memory for the ErrorStack struct if it is uninitialized
```

Comment:
Consider rephrasing this.

Uninitialized can mean several things.

This is just one example that you could use.
```
Allocates an ErrorStack if the deferenced pointer is null.

More details ...
```